.github/workflows/dh-ci-frontend.yml

```diff
@@ -116,8 +113,3 @@ jobs:
           git add ${{ env.output-path }} || cd .
           git commit -m 'feat(dh): generate DataHub API v1 client' || cd .
           git push origin --force-with-lease || cd .
```

Comment:
suggestion: this step only make sense in relation with the one before it. After removing it, it seems like the entire `generate_client` job is not necessary anymore, no?

Reply:
Yes, you're totally right, thanks for looking out!

3e80919db1a3fbc1b0cf7a5483599bf68c4a80ea